src/App.jsx

```diff
@@ -27,7 +27,7 @@ function App() {
             axios.get('https://6327175fba4a9c47533089b9.mockapi.io/cart'),
```

Comment:
It's time to move all requests to the separate file. "apiHelper"
Example: 
const getItems = async () => {
return await axios.get('https://6327175fba4a9c47533089b9.mockapi.io/items'),
}